server/adapters/statistic.js

```diff
@@ -0,0 +1,10 @@
+module.exports = function initStatisticAdapters(models, adapters, libs) {
+  adapters.test = function() {
+
+    libs.statisticRegisterMethodForEntity('testdefct', 'name1', function(){
+      console.log('sucessor');
+    })
+
+    libs.statisticFunctions['testdefct']['name1']();
```

Comment:
@NoSkiiilL @phong-steph 
You should not have any test function in a pull request, because beyond the PR is our **production environment**. Please update it.